app/src/main/kotlin/com/wire/android/ui/home/conversations/MessageItem.kt

```diff
@@ -142,6 +146,23 @@ fun MessageItem(
             Modifier.background(defaultBackgroundColor)
         }
 
+        val colorAnimation = remember { Animatable(Color.Transparent) }
+        val highlightColor = colorsScheme().selectedMessageHighlightColor
+        val transparentColor = colorsScheme().primary.copy(alpha = 0F)
+        LaunchedEffect(isSelectedMessage) {
+            if (isSelectedMessage) {
+                colorAnimation.snapTo(highlightColor)
+                colorAnimation.animateTo(
```

Comment:
suggestion: maybe adding a delay and having the highlight to be visible a little bit longer would make it better because now it disappears quite quickly 😄

Reply:
I did try with a delay on the UI but as we are listening for the changes on `isSelectedMessage` we would also have to increase the delay on the ViewModel, would that also make sense?

The whole appearence of the feature is around 2 seconds (quite compared to current iOS)